.rubocop.yml

```diff
@@ -134,12 +104,6 @@ Style/SymbolArray:
 Style/TernaryParentheses:
   EnforcedStyle: require_parentheses_when_complex
 
-Style/TrailingCommaInArrayLiteral:
-  EnforcedStyleForMultiline: comma
-
-Style/TrailingCommaInHashLiteral:
-  EnforcedStyleForMultiline: comma
-
 #Allow both ['a', 'b'], %w[a b] and %w(a b) style arrays
 Style/WordArray:
   Enabled: false
```

Comment:
Why are you leaving some rules here? What is the key to put it here and there? Can you document this key?

Reply:
TL;DR: I'm not sure what to move to `.rubocop_todo.yml` and what to the gem.

I've moved only those, that I remember Foreman maintainers having disscussion around, others I've left in foreman and I'd like others to tell me if to move them or not. My idea currently is to create issues for them in the gem. But I'll go over them and try gather more info on them prior to that.

Good example is a hash rockets:
It is disabled here, but only because [we've decided](https://community.theforeman.org/t/hash-rocket-syntax/5865) against `rubocop -a` on that rule, but we would like to slowly get there, so the common config should prefer `key: a` style over rockets, so it should move to `.rubocop_todo.yml`.